linked_list_test.py

```diff
@@ -0,0 +1,141 @@
+import io
+import sys
+import pytest
+from linked_list import LinkedList
```

Comment:
[pep 0008 ](https://peps.python.org/pep-0008/#imports)
```suggestion
import io
import sys

import pytest

from linked_list import LinkedList
```